TwitchPlaysAssembly/Src/UI/ModuleCameras.cs

```diff
@@ -183,6 +104,9 @@ private void LateUpdate()
 		{
 			if (Input.GetKey(KeyCode.Escape))
 				EscapePressed = true;
+
+			Module?.UpdateLayerData();
```

Comment:
Module is a MonoBehaviour and so we should do a explicit null check.

Reply:
That won't be necessary, because ModuleCamera doesn't own the module and doesn't use `Destroy` on it. That would make no sense. It sets the field to null explicitly, so this code is fine.

Reply:
Other things do call destroy on a TwitchComponentHandle though. Couldn't that cause a problem? https://github.com/samfun123/KtaneTwitchPlays/blob/master/TwitchPlaysAssembly/Src/MessageResponders/BombMessageResponder.cs#L258-L267